data/json/furniture_and_terrain/furniture-storage.json

```
@@ -433,6 +433,33 @@
       ]
     }
   },
+  {
+    "type": "furniture",
+    "id": "f_clothing_rail",
+    "name": "clothing rail",
+    "description": "A rail for hanging clothes on.",
+    "symbol": "{",
+    "color": "light_gray",
+    "move_cost_mod": -1,
+    "coverage": 10,
+    "required_str": 8,
+    "flags": [ "TRANSPARENT", "FLAMMABLE_HARD", "PLACE_ITEM", "BLOCKSDOOR" ],
+    "deconstruct": { "items": [ { "item": "pipe", "count": [ 4, 6 ] }, { "item": "sheet_metal_small", "count": [ 0, 4 ] } ] }, { "item": "wheel_casters", "count": 1 }
```

```suggestion
      {
    "deconstruct": {
      "items": [
        { "item": "pipe", "count": [ 4, 6 ] },
        { "item": "sheet_metal_small", "count": [ 0, 4 ] },
        { "item": "wheel_casters", "count": 1 }
      ]
    }
  }
```

Will need some linting after this